service/src/java/org/apache/hive/service/cli/thrift/ThriftHttpServlet.java

```diff
@@ -214,9 +214,13 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
             }
           } else if (HiveSamlUtils.isSamlAuthMode(authType)) {
```

Comment:
If `authType` contains `"KERBEROS"`, then this condition block will never be executed. Was the plan to support kerberos and SAML in the same time?

Reply:
so we have had trouble supporting kerberos + other forms of authentication in parallel. So the plan is to NOT support parallel auth forms when it is KERBEROS.
Initially, in http mode, it will be LDAP/SAML/JWT in parallel.